CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## 3.1.0
```

Comment:
This has to be 3.2.0, since 3.1.0 was already released

Reply:
Fixed, see new commit.